membrane_curvature/tests/test_mdakit_membcurv.py

```diff
@@ -3,239 +3,90 @@
 """
 
 # Import package, test suite, and other packages as needed
-import pickle
-
 import pytest
-import sys
-import math
-import os
-import mdtraj as md  # This will be gone after refactoring
 import itertools as it
-from ..lib.mods import core_fast_leaflet, curvature, mean_curvature, gaussian_curvature, dict2pickle
+from ..lib.mods import core_fast_leaflet, curvature, mean_curvature, gaussian_curvature, grid_map
 import numpy as np
 from numpy.testing import assert_almost_equal
 import MDAnalysis as mda
-from membrane_curvature.tests.datafiles import GRO_MEMBRANE_PROTEIN, XTC_MEMBRANE_PROTEIN, GRO_PO4, XTC_PO4
+from membrane_curvature.tests.datafiles import (GRO_PO4_SMALL, XTC_PO4_SMALL)
 
 # Reference data from datafile
 MEMBRANE_CURVATURE_DATA = {
-    'mean_curvature': np.array([[0.06396822, 0.2254189, 0.22399141, 0.02498465, 0.05090628,
```

Comment:
Why are you removing/changing tests here? This seems dangerous. How do you know that your changes did not introduce bugs? 

For this PR I would have expected _no changes_ to the tests.

Reply:
I will put them back, I just thought of simplifying the tests by providing smaller arrays. But I see your point.

Reply:
Previous tests`test_gaussian_curvature` and `test_mean_curvature`(now called `test_gaussian_curvature_all`and `test_mean_curvature_all`, repectively) added back and passing.